modules/elastodyn/src/ElastoDyn.f90

```diff
@@ -4440,229 +4463,10 @@ SUBROUTINE SetOutParam(OutList, p, ErrStat, ErrMsg )
 
 
 !   ..... Developer must add checking for invalid inputs here: .....
-if (p%BD4Blades) then
```

Comment:
This block of code is missing.  We'll need to put it back in.

Reply:
Resolved with commit "7303f6cc5"